src/main/java/seedu/address/logic/parser/ListCommandParser.java

```diff
@@ -24,24 +29,27 @@ public ListCommand parse(String args) throws ParseException {
         String sortingAttribute = argMultimap.getValue(CliSyntax.PREFIX_SORT).orElse(null);
         System.out.println("sortingAttribute: " + sortingAttribute);
```

Comment:
Do remove instances of `System.out.println`!

Reply:
Ok! 😄